src/main/java/uk/gov/hmcts/sscs/controller/AppealsController.java

```diff
@@ -72,6 +75,17 @@ public AppealsController(TribunalsService tribunalsService, SubmitAppealService
         return ok(tribunalsService.findAppeal(appealNumber).toString());
     }
 
+    @ApiOperation(value = "unsubscribe", notes = "Removes subscription and returns benefit type in the response json",
+            response = String.class, responseContainer = "Unsubscribed appeal benefit type")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "Removed subscription", response = String.class)})
+    @ResponseBody
+    @RequestMapping(value = "/appeals/{appealNumber}/subscribe/reason/{reason}", method = DELETE, produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> unsubscribe(@PathVariable String appealNumber, @PathVariable String reason)
+            throws CcdException {
+        String benefitType = tribunalsService.unsubscribe(appealNumber, reason);
```

Comment:
Does this make sense to call the `tribunalsService`? Or can it just call `submitAppealService` directly?

Reply:
Not sure. Aaron might know more about this. He created Jira ticket so that means i guess it is required. I will leave it for now.